src/avfoundation.cs

```diff
@@ -2507,6 +2789,9 @@ interface AVAsset : NSCopying {
 		[Export ("metadataForFormat:")]
 		AVMetadataItem [] MetadataForFormat (string format);
 
+		[Wrap ("MetadataForFormat (format.GetConstant ())")]
+		AVMetadataItem [] MetadataForFormat (AVMetadataFormat format);
```

Comment:
Should the same be done for TracksWithMediaCharacteristic for example? Since the same issue occurs.